source/feed.liquid

```diff
@@ -0,0 +1,34 @@
+---
+pagination:
+  data: urls
+  alias: url
+  size: 1
+urls:
+  - 'feed.xml'
+  - 'feed'
+permalink: '{{ url }}'
+eleventyExcludeFromCollections: true
+---
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+  <title>Sass Blog</title>
+  <id>{{ '/blog' | absoluteUrl: site.url }}</id>
+  <link href="{{ '/blog' | absoluteUrl: site.url }}"/>
+  <link href="{{ '/feed.xml' | absoluteUrl: site.url }}" rel="self"/>
+  <updated>{{ collections.blog | getNewestCollectionItemDate | dateToRfc3339 }}</updated>
+  {%- assign posts = collections.blog | reverse -%}
+  {%- for post in posts limit:6 %}
+  {%- assign absolutePostUrl = post.url | absoluteUrl: site.url -%}
+  <entry>
+    <title>{{ post.data.title }}</title>
+    <link href="{{ absolutePostUrl }}" rel="alternate"/>
+    <id>{{ absolutePostUrl }}</id>
+    <published>{{ post.date | dateToRfc3339 }}</published>
+    <updated>{{ post.data.updated | default: post.date | dateToRfc3339 }}</updated>
```

Comment:
The existing site uses the file modified date for this `<updated>` tag. From reading the Atom spec, I think that might unnecessary in most cases (it's only supposed to update for substantial changes), and I'd be nervous about trusting file creation/modified timestamps on a static site. I think this is a better and more explicit solution (an optional `updated` timestamp on the blog post), but we can run it by Natalie.